examples/demo-nextflow-project/agc-project.yaml

```diff
@@ -26,3 +26,6 @@ contexts:
     engines:
       - type: nextflow
         engine: nextflow
+        resourceRequirements:
+            vcpus: 1
```

Comment:
Nit - this may as well be 2 as the smallest instance size will have 2CPU

Reply:
Thanks, I updated it to 2 as suggested.